README.md

```diff
@@ -31,8 +31,8 @@ For an overview of all the components in the gfx-rs ecosystem, see [the big pict
 
 ### MSRV policy
 
-Minimum Supported Rust Version is **1.64**.
-It is enforced on CI (in "/.github/workflows/ci.yml") with `RUST_VERSION` variable.
+Minimum Supported Rust Version is **1.65**.
+It is enforced on CI (in "/.github/workflows/ci.yml", "/.github/workflows/cts.yml") with `RUST_VERSION` variable.
```

Comment:
I wouldn't bother mentioning this as the cts file is likely going to go away soon.

Reply:
Reverted this hunk with c14908b5c9c53479a2c6433d3ac5dadd7c85250a.